tests/include/log.hpp

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
1. Move mocks folder to `tests/iam`
2. We don't need separate `include` dir, all headers should be with corresponding cpp files.
3. Move `log.hpp` to utils.

Reply:
1. As the new integration tests include all test scenarious from the old unit tests I removed the latter together with required mocks.
2/3. done